src/installer/installer.go

```diff
@@ -586,6 +586,11 @@ func (i *installer) waitForMasterNodes(ctx context.Context, minMasterNodes int,
 			i.log.Warnf("Still waiting for master nodes: %v", err)
 			return false
 		}
+		for _, node := range nodes.Items {
+			if err = kc.UntaintNode(node.Name); err != nil {
+				i.log.Warnf("Failed to untaint node %s: %v", node.Name, err)
+			}
+		}
```

Comment:
I think we want to do this only for not Ready master nodes.
* We are checking if they are ready in the next function call